packages/utils/src/isEqual.ts

```diff
@@ -1,9 +1,9 @@
 import { isPlainObject } from "./isPlainObject"
 
 /**
- * Compares two values for equality.
+ * Compares two values for equality ignoring specified fields when provided.
  */
-export function isEqual(a: unknown, b: unknown): boolean {
+export function isEqual(a: unknown, b: unknown, ignoreFields: string[] = []): boolean {
```

Comment:
are the ignored fields applied at every level?

Reply:
It's applied at every level for object data

Reply:
no need to modify isEqual
just create a cache key that doesn't require this

Reply:
what if you want to ignore specific fields at the top but other fields at lower levels?